move_base/src/move_base.cpp

```diff
@@ -45,6 +45,7 @@
 #include <geometry_msgs/Twist.h>
 
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <move_base/Path.h>
```

Comment:
we should not be adding messages to move_base - there is a move_base_msgs package in a separate repo specifically so that users of nav do not have to build all of nav on remote machines that are monitoring outputs.

There is also a standard path message in nav_msgs, why not use that?

Reply:
Hi,
I used my msg instead of nav_msg/Path, since I need

float32 dist_to_goal # Path distance left to goal (m)
float32 time_to_goal # Time left to goal to navigate along path (s)